### ISSUE TITLE
hossted set auth [appName] true

### ISSUE THREAD

**Issue by billylkc** [hossted/cli#32]
Separated ticket to handle **set auth [appName] true**

@niryariv, Please help with the following
- Provide a testing machine (e.g. gitbucket)
- Provide exact steps/lines to comment/uncomment (instead of the sed commands from #6)
- Provide a checking logic if the docker file does not contains those lines (i.e. which lines to check for error, say wordpress is not applicable for the commands, what exact lines I should check in the docker files)

**Comment by billylkc** [comment]
It would be a great help if you can send over/attach the docker-compose file for reference as well

**Comment by niryariv** [comment]
@liorkesos please help with Billy's requests above.

**Comment by niryariv** [comment]
@billylkc test machine available at 3.215.23.221, sent key file in email

**Comment by liorkesos** [comment]
We basically need to toggle with a comment the settings you are [setting here](https://github.com/hossted/cli/blob/d66af346a2595dd1ef6e6b25030dfae4c85e4b33/hossted/setAuth.go#L40)...
```
gitbucket -> label
  - "traefik.http.middlewares.tauth.basicauth.usersfile=letsencrypt/.htpass"
  - "traefik.http.routers.$PROJECT_NAME.middlewares=tauth"
```
